log.go

```diff
@@ -142,17 +146,30 @@ func (log *Log) Trace(message ...interface{}) {
 }
 
 func Trace(message ...interface{}) {
-	std.Trace(message)
+	std.Trace(message...)
 }
 
 func (log *Log) Println(message ...interface{}) {
-	log.rotateLog()
-
-	internal_logger.Println(message)
+	pidStr := "[" + strconv.Itoa(os.Getpid()) + "]"
+
+	if firstArg, isStr := message[0].(string); isStr {
+		// Passing the empty slice for the arguments works here, so no need to check len(message)
+		internal_logger.Printf(pidStr+" "+firstArg, message[1:]...)
```

Comment:
Would this read better as:
```
internal_logger.Printf(fmt.Sprintf("[%d] %s", os.GetPid(), firstArg), message[1:]...)
```
And then below:
```
internal_logger.Printf("[%d] Unsupported argument type in first log argument, called from the line above %s:%d", os.GetPid(), fileName, line)
```
I get the DRY, but it makes it harder to read, and we still only get the PID once, its just in two places.